app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -26,11 +26,14 @@ def perform(_now)
     InPersonEnrollment.needs_usps_status_check(...5.minutes.ago).each do |enrollment|
       # Record and commit attempt to check enrollment status to database
       enrollment.update(status_check_attempted_at: Time.zone.now)
-      unique_id = enrollment.usps_unique_id
+
+      if enrollment.unique_id.blank?
+        enrollment.update(unique_id: enrollment.usps_unique_id)
+      end
```

Comment:
Is this something we'd want to do once as a backfill, instead of doing it every time we run this cron job?

Reply:
It should only need to be done once, moving forward a unique id will be assigned. This is moreso for cleanup of those that were not assigned a unique id.

Reply:
Yup! But I mean, running a single backfill (sometimes as a migration class) to fix all of them, and then this job doesn't need to check each time?

Reply:
I prefer to not do data migrations in db migrations, probably influenced by [this article](https://thoughtbot.com/blog/data-migrations-in-rails). My practice in the past has been to (as described in that article) create one-off rake tasks that can be run to perform the migration.

In this case, having this conditional catch all the enrollments with old-style IDs felt like the most straightforward way, and it doesn't require any coordination (eg making sure the db migration has run, starting a console on the environment, running the data migration, etc.). Either way we'd have to have a follow-up PR to clean up the in-between state.